content/articles/ordering-lets-encrypt-certificate.markdown

```diff
@@ -58,6 +58,20 @@ The order is the first step into getting an SSL certificate. It will create an S
     1.  Submit the order.
 </div>
 
+### What this looks like
+
+As mentioned above, you will see a different form for ordering a Let's Encrypt certificate based on the plan to which you are subscribed. If you have full access to SAN certificate configuration, this is what you'll see:
+
+![Let's Encrypt with SAN](/files/lets-encrypt-new-san.png)
+
+On a less advanced plans you will see the ability to provide only one name on your Let's Encrypt certificate:
```

Comment:
> less advanced plans

I actually don't really like this classification. I'd simply use Personal/Professional/Business for now.